returnn/training.py

```diff
@@ -237,6 +238,48 @@ def _get_run_cmd(self):
 
         return run_cmd
 
+    def info(self):
+        def try_load_lr_log(files: List[str]):
+            # Used in parsing the learning rates
+            @dataclass
+            class EpochData:
+                learningRate: float
+                error: Dict[str, float]
+
+            for file in files:
+                try:
+                    with open(file, "rt") as file:
+                        return eval(file.read().strip())
+                except FileNotFoundError:
+                    pass
+
+            return None
+
+        lr_during_training = os.path.join(
+            self._sis_path(gs.JOB_WORK_DIR),
+            self.returnn_config.get("learning_rate_file", "learning_rates"),
+        )
+        lr_after_training = tk.uncached_path(self.out_learning_rates)
```

Comment:
This should not be required, since the LR file will stay in the work folder even if training is finished.

Reply:
also don't use `uncached_path` but instead `.get_path()`

Reply:
> This should not be required, since the LR file will stay in the work folder even if training is finished.

There exists an option in sisyphus to clean up the work folder after the training is finished. If someone had this activated, then the `lr_after_training` would be necessary.

But +1 for 
```suggestion
        lr_after_training = self.out_learning_rates.get_path()
```

Reply:
> > This should not be required, since the LR file will stay in the work folder even if training is finished.
> 
> There exists an option in sisyphus to clean up the work folder after the training is finished. If someone had this activated, then the `lr_after_training` would be necessary.

To my understanding `info` will only ever be called when the job is running, so if it is finished and cleaned, we do not need this function anyway.